annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AdminServiceImpl.java

```diff
@@ -140,12 +128,11 @@ public BatchProcessingStatus reindexAnnotationSet(List<String> ids, boolean isOb
 						if (isObjectId) {
```

Comment:
this should not be needed anymore, we should use only the identifiers or the anno_uris